phpunit.xml

```diff
@@ -10,21 +10,24 @@
 	failOnWarning="true"
 	verbose="true">
 	<php>
-		<const
-			name="OVERRIDE_MYSQL_CONFIG"
+		<env
+			name="MYSQL_CONFIG_FROM_ENVIRONMENT"
```

Comment:
Yeah we could have a separate `.env` file just for tests. Like `test.env`, or something. I would still need an environment variable to dictate which file to choose to parse, though. SO there would still need to be some switching happening. In a future version, I plan to use dependency injection of properties like this so a lot of this can actually go away. It's out of scope for this ticket, so I have these changes stashed away in another branch.

Reply:
Resolved in 649c9fafbbb6fe41e824131ba36b185b0cf448d3